README.md

```diff
@@ -12,10 +12,116 @@
 
 ## Installation
 
+To install the SignalFlow Python client library, open a terminal and run the
+following command:
+
 ```console
 pip install signalflow-client-python
 ```
 
+## Run a SignalFlow computation
+
+The following example allows you run a SignalFlow computation from the command
+line.
+
+1. Create a file called ``signalflow_demo.py`` that includes the following
+   content:
+
+   ```python
+   #!/usr/bin/env python
```

Comment:
In the interest of stripping away anything unnecessary, can we remove this license?